New log file opened at Tue Apr 05 17:20:33 CEST 2016 by 2620@killermenplpc
Taking next connection to: 53824
New client attempting to connect on port: 53827-53824
  Address = /192.168.56.1:53824
This client on port 53827 has uuid 97508552-879c-4ff2-ada1-076c14b5a16e configName activator humanReadableName Activator protocolVersion 3 featureTags Vector()
Connected Clients: activator-97508552-879c-4ff2-ada1-076c14b5a16e
Taking next connection to: 53824
New client attempting to connect on port: 53828-53824
  Address = /192.168.56.1:53824
Handshake exception on socket: 53828
sbt.impl.ipc.HandshakeException: Unable to perform handshake
	at sbt.impl.ipc.Peer.handshake(IPC.scala:46)
	at sbt.impl.ipc.MultiClientServer.<init>(IPC.scala:150)
	at sbt.server.SbtServerSocketHandler$$anon$1.run(SbtServerSocketHandler.scala:43)
Caused by: java.net.SocketException: Software caused connection abort: recv failed
	at java.net.SocketInputStream.socketRead0(Native Method)
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:116)
	at java.net.SocketInputStream.read(SocketInputStream.java:170)
	at java.net.SocketInputStream.read(SocketInputStream.java:141)
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246)
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265)
	at java.io.DataInputStream.readInt(DataInputStream.java:387)
	at sbt.impl.ipc.Peer.receive(IPC.scala:95)
	at sbt.impl.ipc.Peer.handshake(IPC.scala:49)
	... 2 more
Taking next connection to: 53824
New client attempting to connect on port: 53829-53824
  Address = /192.168.56.1:53824
This client on port 53829 has uuid 44fc7ba3-7c3c-47c2-8ee4-e0bb0c2f1386 configName activator humanReadableName Activator protocolVersion 3 featureTags Vector()
Connected Clients: activator-97508552-879c-4ff2-ada1-076c14b5a16e, activator-44fc7ba3-7c3c-47c2-8ee4-e0bb0c2f1386
Taking next connection to: 53824
New client attempting to connect on port: 53942-53824
  Address = /192.168.56.1:53824
Handshake exception on socket: 53942
sbt.impl.ipc.HandshakeException: Unable to perform handshake
	at sbt.impl.ipc.Peer.handshake(IPC.scala:46)
	at sbt.impl.ipc.MultiClientServer.<init>(IPC.scala:150)
	at sbt.server.SbtServerSocketHandler$$anon$1.run(SbtServerSocketHandler.scala:43)
Caused by: java.net.SocketException: Software caused connection abort: recv failed
	at java.net.SocketInputStream.socketRead0(Native Method)
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:116)
	at java.net.SocketInputStream.read(SocketInputStream.java:170)
	at java.net.SocketInputStream.read(SocketInputStream.java:141)
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246)
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265)
	at java.io.DataInputStream.readInt(DataInputStream.java:387)
	at sbt.impl.ipc.Peer.receive(IPC.scala:95)
	at sbt.impl.ipc.Peer.handshake(IPC.scala:49)
	... 2 more
Taking next connection to: 53824
New client attempting to connect on port: 53943-53824
  Address = /192.168.56.1:53824
This client on port 53943 has uuid 54df02df-3de8-43eb-8e17-2c1ff5057c69 configName play-fork-run humanReadableName Play Fork Run protocolVersion 1 featureTags Vector()
Connected Clients: activator-44fc7ba3-7c3c-47c2-8ee4-e0bb0c2f1386, play-fork-run-54df02df-3de8-43eb-8e17-2c1ff5057c69
Taking next connection to: 53824
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 53824
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 53824
New client attempting to connect on port: 54513-53824
  Address = /192.168.56.1:53824
Handshake exception on socket: 54513
sbt.impl.ipc.HandshakeException: Unable to perform handshake
	at sbt.impl.ipc.Peer.handshake(IPC.scala:46)
	at sbt.impl.ipc.MultiClientServer.<init>(IPC.scala:150)
	at sbt.server.SbtServerSocketHandler$$anon$1.run(SbtServerSocketHandler.scala:43)
Caused by: java.net.SocketException: Software caused connection abort: recv failed
	at java.net.SocketInputStream.socketRead0(Native Method)
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:116)
	at java.net.SocketInputStream.read(SocketInputStream.java:170)
	at java.net.SocketInputStream.read(SocketInputStream.java:141)
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246)
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265)
	at java.io.DataInputStream.readInt(DataInputStream.java:387)
	at sbt.impl.ipc.Peer.receive(IPC.scala:95)
	at sbt.impl.ipc.Peer.handshake(IPC.scala:49)
	... 2 more
Taking next connection to: 53824
New client attempting to connect on port: 54514-53824
  Address = /192.168.56.1:53824
This client on port 54514 has uuid dc6bf797-a262-432f-a20e-d499483f1d82 configName play-fork-run humanReadableName Play Fork Run protocolVersion 1 featureTags Vector()
Connected Clients: activator-44fc7ba3-7c3c-47c2-8ee4-e0bb0c2f1386, play-fork-run-dc6bf797-a262-432f-a20e-d499483f1d82
Taking next connection to: 53824
New client attempting to connect on port: 54565-53824
  Address = /192.168.56.1:53824
Handshake exception on socket: 54565
sbt.impl.ipc.HandshakeException: Unable to perform handshake
	at sbt.impl.ipc.Peer.handshake(IPC.scala:46)
	at sbt.impl.ipc.MultiClientServer.<init>(IPC.scala:150)
	at sbt.server.SbtServerSocketHandler$$anon$1.run(SbtServerSocketHandler.scala:43)
Caused by: java.net.SocketException: Software caused connection abort: recv failed
	at java.net.SocketInputStream.socketRead0(Native Method)
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:116)
	at java.net.SocketInputStream.read(SocketInputStream.java:170)
	at java.net.SocketInputStream.read(SocketInputStream.java:141)
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246)
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265)
	at java.io.DataInputStream.readInt(DataInputStream.java:387)
	at sbt.impl.ipc.Peer.receive(IPC.scala:95)
	at sbt.impl.ipc.Peer.handshake(IPC.scala:49)
	... 2 more
Taking next connection to: 53824
New client attempting to connect on port: 54566-53824
  Address = /192.168.56.1:53824
This client on port 54566 has uuid ad34020d-5925-4d59-9390-b9c493e1005a configName play-fork-run humanReadableName Play Fork Run protocolVersion 1 featureTags Vector()
Connected Clients: activator-44fc7ba3-7c3c-47c2-8ee4-e0bb0c2f1386, play-fork-run-ad34020d-5925-4d59-9390-b9c493e1005a
Taking next connection to: 53824
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 53824
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 53824
New client attempting to connect on port: 55098-53824
  Address = /192.168.56.1:53824
Handshake exception on socket: 55098
sbt.impl.ipc.HandshakeException: Unable to perform handshake
	at sbt.impl.ipc.Peer.handshake(IPC.scala:46)
	at sbt.impl.ipc.MultiClientServer.<init>(IPC.scala:150)
	at sbt.server.SbtServerSocketHandler$$anon$1.run(SbtServerSocketHandler.scala:43)
Caused by: java.net.SocketException: Software caused connection abort: recv failed
	at java.net.SocketInputStream.socketRead0(Native Method)
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:116)
	at java.net.SocketInputStream.read(SocketInputStream.java:170)
	at java.net.SocketInputStream.read(SocketInputStream.java:141)
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246)
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265)
	at java.io.DataInputStream.readInt(DataInputStream.java:387)
	at sbt.impl.ipc.Peer.receive(IPC.scala:95)
	at sbt.impl.ipc.Peer.handshake(IPC.scala:49)
	... 2 more
Taking next connection to: 53824
New client attempting to connect on port: 55099-53824
  Address = /192.168.56.1:53824
This client on port 55099 has uuid 405863f7-a19b-4f48-9850-ae4e333210d3 configName play-fork-run humanReadableName Play Fork Run protocolVersion 1 featureTags Vector()
Connected Clients: activator-44fc7ba3-7c3c-47c2-8ee4-e0bb0c2f1386, play-fork-run-405863f7-a19b-4f48-9850-ae4e333210d3
Taking next connection to: 53824
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 53824
New client attempting to connect on port: 55483-53824
  Address = /192.168.56.1:53824
Handshake exception on socket: 55483
sbt.impl.ipc.HandshakeException: Unable to perform handshake
	at sbt.impl.ipc.Peer.handshake(IPC.scala:46)
	at sbt.impl.ipc.MultiClientServer.<init>(IPC.scala:150)
	at sbt.server.SbtServerSocketHandler$$anon$1.run(SbtServerSocketHandler.scala:43)
Caused by: java.net.SocketException: Software caused connection abort: recv failed
	at java.net.SocketInputStream.socketRead0(Native Method)
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:116)
	at java.net.SocketInputStream.read(SocketInputStream.java:170)
	at java.net.SocketInputStream.read(SocketInputStream.java:141)
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246)
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265)
	at java.io.DataInputStream.readInt(DataInputStream.java:387)
	at sbt.impl.ipc.Peer.receive(IPC.scala:95)
	at sbt.impl.ipc.Peer.handshake(IPC.scala:49)
	... 2 more
Taking next connection to: 53824
New client attempting to connect on port: 55484-53824
  Address = /192.168.56.1:53824
This client on port 55484 has uuid 0c16d668-d338-461e-93ae-09072700c2e6 configName play-fork-run humanReadableName Play Fork Run protocolVersion 1 featureTags Vector()
Connected Clients: activator-44fc7ba3-7c3c-47c2-8ee4-e0bb0c2f1386, play-fork-run-0c16d668-d338-461e-93ae-09072700c2e6
Taking next connection to: 53824
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 53824
New client attempting to connect on port: 55920-53824
  Address = /192.168.56.1:53824
Handshake exception on socket: 55920
sbt.impl.ipc.HandshakeException: Unable to perform handshake
	at sbt.impl.ipc.Peer.handshake(IPC.scala:46)
	at sbt.impl.ipc.MultiClientServer.<init>(IPC.scala:150)
	at sbt.server.SbtServerSocketHandler$$anon$1.run(SbtServerSocketHandler.scala:43)
Caused by: java.net.SocketException: Software caused connection abort: recv failed
	at java.net.SocketInputStream.socketRead0(Native Method)
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:116)
	at java.net.SocketInputStream.read(SocketInputStream.java:170)
	at java.net.SocketInputStream.read(SocketInputStream.java:141)
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246)
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265)
	at java.io.DataInputStream.readInt(DataInputStream.java:387)
	at sbt.impl.ipc.Peer.receive(IPC.scala:95)
	at sbt.impl.ipc.Peer.handshake(IPC.scala:49)
	... 2 more
Taking next connection to: 53824
New client attempting to connect on port: 55921-53824
  Address = /192.168.56.1:53824
This client on port 55921 has uuid 768860fd-f41e-4b3f-8877-91b294e74929 configName play-fork-run humanReadableName Play Fork Run protocolVersion 1 featureTags Vector()
Connected Clients: activator-44fc7ba3-7c3c-47c2-8ee4-e0bb0c2f1386, play-fork-run-768860fd-f41e-4b3f-8877-91b294e74929
Taking next connection to: 53824
New client attempting to connect on port: 56077-53824
  Address = /192.168.56.1:53824
Handshake exception on socket: 56077
sbt.impl.ipc.HandshakeException: Unable to perform handshake
	at sbt.impl.ipc.Peer.handshake(IPC.scala:46)
	at sbt.impl.ipc.MultiClientServer.<init>(IPC.scala:150)
	at sbt.server.SbtServerSocketHandler$$anon$1.run(SbtServerSocketHandler.scala:43)
Caused by: java.net.SocketException: Software caused connection abort: recv failed
	at java.net.SocketInputStream.socketRead0(Native Method)
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:116)
	at java.net.SocketInputStream.read(SocketInputStream.java:170)
	at java.net.SocketInputStream.read(SocketInputStream.java:141)
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246)
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265)
	at java.io.DataInputStream.readInt(DataInputStream.java:387)
	at sbt.impl.ipc.Peer.receive(IPC.scala:95)
	at sbt.impl.ipc.Peer.handshake(IPC.scala:49)
	... 2 more
Taking next connection to: 53824
New client attempting to connect on port: 56078-53824
  Address = /192.168.56.1:53824
This client on port 56078 has uuid b7e583ff-7ae1-4538-aad3-5f5177ceadc8 configName play-fork-run humanReadableName Play Fork Run protocolVersion 1 featureTags Vector()
Connected Clients: activator-44fc7ba3-7c3c-47c2-8ee4-e0bb0c2f1386, play-fork-run-b7e583ff-7ae1-4538-aad3-5f5177ceadc8
Taking next connection to: 53824
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 53824
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 53824
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 53824
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 53824
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 53824
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 53824
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 53824
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 53824
New client attempting to connect on port: 58092-53824
  Address = /192.168.56.1:53824
Handshake exception on socket: 58092
sbt.impl.ipc.HandshakeException: Unable to perform handshake
	at sbt.impl.ipc.Peer.handshake(IPC.scala:46)
	at sbt.impl.ipc.MultiClientServer.<init>(IPC.scala:150)
	at sbt.server.SbtServerSocketHandler$$anon$1.run(SbtServerSocketHandler.scala:43)
Caused by: java.net.SocketException: Software caused connection abort: recv failed
	at java.net.SocketInputStream.socketRead0(Native Method)
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:116)
	at java.net.SocketInputStream.read(SocketInputStream.java:170)
	at java.net.SocketInputStream.read(SocketInputStream.java:141)
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246)
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265)
	at java.io.DataInputStream.readInt(DataInputStream.java:387)
	at sbt.impl.ipc.Peer.receive(IPC.scala:95)
	at sbt.impl.ipc.Peer.handshake(IPC.scala:49)
	... 2 more
Taking next connection to: 53824
New client attempting to connect on port: 58093-53824
  Address = /192.168.56.1:53824
This client on port 58093 has uuid 56b390e6-f5b0-4e94-a1c0-a79901fc0b52 configName play-fork-run humanReadableName Play Fork Run protocolVersion 1 featureTags Vector()
Connected Clients: activator-44fc7ba3-7c3c-47c2-8ee4-e0bb0c2f1386, play-fork-run-56b390e6-f5b0-4e94-a1c0-a79901fc0b52
Taking next connection to: 53824
New client attempting to connect on port: 58135-53824
  Address = /192.168.56.1:53824
Handshake exception on socket: 58135
sbt.impl.ipc.HandshakeException: Unable to perform handshake
	at sbt.impl.ipc.Peer.handshake(IPC.scala:46)
	at sbt.impl.ipc.MultiClientServer.<init>(IPC.scala:150)
	at sbt.server.SbtServerSocketHandler$$anon$1.run(SbtServerSocketHandler.scala:43)
Caused by: java.net.SocketException: Software caused connection abort: recv failed
	at java.net.SocketInputStream.socketRead0(Native Method)
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:116)
	at java.net.SocketInputStream.read(SocketInputStream.java:170)
	at java.net.SocketInputStream.read(SocketInputStream.java:141)
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246)
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265)
	at java.io.DataInputStream.readInt(DataInputStream.java:387)
	at sbt.impl.ipc.Peer.receive(IPC.scala:95)
	at sbt.impl.ipc.Peer.handshake(IPC.scala:49)
	... 2 more
Taking next connection to: 53824
New client attempting to connect on port: 58136-53824
  Address = /192.168.56.1:53824
This client on port 58136 has uuid 62b07f0d-df21-4cf7-a7a7-1bf2a1b3ad97 configName play-fork-run humanReadableName Play Fork Run protocolVersion 1 featureTags Vector()
Connected Clients: activator-44fc7ba3-7c3c-47c2-8ee4-e0bb0c2f1386, play-fork-run-62b07f0d-df21-4cf7-a7a7-1bf2a1b3ad97
Taking next connection to: 53824
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 53824
New client attempting to connect on port: 58447-53824
  Address = /192.168.56.1:53824
Handshake exception on socket: 58447
sbt.impl.ipc.HandshakeException: Unable to perform handshake
	at sbt.impl.ipc.Peer.handshake(IPC.scala:46)
	at sbt.impl.ipc.MultiClientServer.<init>(IPC.scala:150)
	at sbt.server.SbtServerSocketHandler$$anon$1.run(SbtServerSocketHandler.scala:43)
Caused by: java.net.SocketException: Software caused connection abort: recv failed
	at java.net.SocketInputStream.socketRead0(Native Method)
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:116)
	at java.net.SocketInputStream.read(SocketInputStream.java:170)
	at java.net.SocketInputStream.read(SocketInputStream.java:141)
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246)
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265)
	at java.io.DataInputStream.readInt(DataInputStream.java:387)
	at sbt.impl.ipc.Peer.receive(IPC.scala:95)
	at sbt.impl.ipc.Peer.handshake(IPC.scala:49)
	... 2 more
Taking next connection to: 53824
New client attempting to connect on port: 58448-53824
  Address = /192.168.56.1:53824
This client on port 58448 has uuid 857cba3f-7705-417d-83b8-ff88425a5769 configName play-fork-run humanReadableName Play Fork Run protocolVersion 1 featureTags Vector()
Connected Clients: activator-44fc7ba3-7c3c-47c2-8ee4-e0bb0c2f1386, play-fork-run-62b07f0d-df21-4cf7-a7a7-1bf2a1b3ad97, play-fork-run-857cba3f-7705-417d-83b8-ff88425a5769
Taking next connection to: 53824
New client attempting to connect on port: 58552-53824
  Address = /192.168.56.1:53824
Handshake exception on socket: 58552
sbt.impl.ipc.HandshakeException: Unable to perform handshake
	at sbt.impl.ipc.Peer.handshake(IPC.scala:46)
	at sbt.impl.ipc.MultiClientServer.<init>(IPC.scala:150)
	at sbt.server.SbtServerSocketHandler$$anon$1.run(SbtServerSocketHandler.scala:43)
Caused by: java.io.EOFException
	at java.io.DataInputStream.readInt(DataInputStream.java:392)
	at sbt.impl.ipc.Peer.receive(IPC.scala:95)
	at sbt.impl.ipc.Peer.handshake(IPC.scala:49)
	... 2 more
Taking next connection to: 53824
New client attempting to connect on port: 58553-53824
  Address = /192.168.56.1:53824
This client on port 58553 has uuid 344415da-d3ff-49b4-96c2-006140b6264a configName play-fork-run humanReadableName Play Fork Run protocolVersion 1 featureTags Vector()
Connected Clients: activator-44fc7ba3-7c3c-47c2-8ee4-e0bb0c2f1386, play-fork-run-344415da-d3ff-49b4-96c2-006140b6264a
Taking next connection to: 53824
New client attempting to connect on port: 58568-53824
  Address = /192.168.56.1:53824
Handshake exception on socket: 58568
sbt.impl.ipc.HandshakeException: Unable to perform handshake
	at sbt.impl.ipc.Peer.handshake(IPC.scala:46)
	at sbt.impl.ipc.MultiClientServer.<init>(IPC.scala:150)
	at sbt.server.SbtServerSocketHandler$$anon$1.run(SbtServerSocketHandler.scala:43)
Caused by: java.net.SocketException: Software caused connection abort: recv failed
	at java.net.SocketInputStream.socketRead0(Native Method)
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:116)
	at java.net.SocketInputStream.read(SocketInputStream.java:170)
	at java.net.SocketInputStream.read(SocketInputStream.java:141)
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246)
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265)
	at java.io.DataInputStream.readInt(DataInputStream.java:387)
	at sbt.impl.ipc.Peer.receive(IPC.scala:95)
	at sbt.impl.ipc.Peer.handshake(IPC.scala:49)
	... 2 more
Taking next connection to: 53824
New client attempting to connect on port: 58569-53824
  Address = /192.168.56.1:53824
This client on port 58569 has uuid 0f0bd611-30ed-48d6-99ad-08daae1b432c configName play-fork-run humanReadableName Play Fork Run protocolVersion 1 featureTags Vector()
Connected Clients: activator-44fc7ba3-7c3c-47c2-8ee4-e0bb0c2f1386, play-fork-run-344415da-d3ff-49b4-96c2-006140b6264a, play-fork-run-0f0bd611-30ed-48d6-99ad-08daae1b432c
Taking next connection to: 53824
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 53824
New client attempting to connect on port: 58901-53824
  Address = /192.168.56.1:53824
Handshake exception on socket: 58901
sbt.impl.ipc.HandshakeException: Unable to perform handshake
	at sbt.impl.ipc.Peer.handshake(IPC.scala:46)
	at sbt.impl.ipc.MultiClientServer.<init>(IPC.scala:150)
	at sbt.server.SbtServerSocketHandler$$anon$1.run(SbtServerSocketHandler.scala:43)
Caused by: java.io.EOFException
	at java.io.DataInputStream.readInt(DataInputStream.java:392)
	at sbt.impl.ipc.Peer.receive(IPC.scala:95)
	at sbt.impl.ipc.Peer.handshake(IPC.scala:49)
	... 2 more
Taking next connection to: 53824
New client attempting to connect on port: 58902-53824
  Address = /192.168.56.1:53824
This client on port 58902 has uuid e95dc4cc-6041-475e-8870-ed3d4dd6ea0d configName play-fork-run humanReadableName Play Fork Run protocolVersion 1 featureTags Vector()
Connected Clients: activator-44fc7ba3-7c3c-47c2-8ee4-e0bb0c2f1386, play-fork-run-e95dc4cc-6041-475e-8870-ed3d4dd6ea0d
Taking next connection to: 53824
New client attempting to connect on port: 59088-53824
  Address = /192.168.56.1:53824
Handshake exception on socket: 59088
sbt.impl.ipc.HandshakeException: Unable to perform handshake
	at sbt.impl.ipc.Peer.handshake(IPC.scala:46)
	at sbt.impl.ipc.MultiClientServer.<init>(IPC.scala:150)
	at sbt.server.SbtServerSocketHandler$$anon$1.run(SbtServerSocketHandler.scala:43)
Caused by: java.net.SocketException: Software caused connection abort: recv failed
	at java.net.SocketInputStream.socketRead0(Native Method)
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:116)
	at java.net.SocketInputStream.read(SocketInputStream.java:170)
	at java.net.SocketInputStream.read(SocketInputStream.java:141)
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246)
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265)
	at java.io.DataInputStream.readInt(DataInputStream.java:387)
	at sbt.impl.ipc.Peer.receive(IPC.scala:95)
	at sbt.impl.ipc.Peer.handshake(IPC.scala:49)
	... 2 more
Taking next connection to: 53824
New client attempting to connect on port: 59089-53824
  Address = /192.168.56.1:53824
This client on port 59089 has uuid f81ecc89-da96-4bbb-a211-7a4e37d3b21d configName play-fork-run humanReadableName Play Fork Run protocolVersion 1 featureTags Vector()
Connected Clients: activator-44fc7ba3-7c3c-47c2-8ee4-e0bb0c2f1386, play-fork-run-f81ecc89-da96-4bbb-a211-7a4e37d3b21d
Taking next connection to: 53824
New client attempting to connect on port: 59132-53824
  Address = /192.168.56.1:53824
Handshake exception on socket: 59132
sbt.impl.ipc.HandshakeException: Unable to perform handshake
	at sbt.impl.ipc.Peer.handshake(IPC.scala:46)
	at sbt.impl.ipc.MultiClientServer.<init>(IPC.scala:150)
	at sbt.server.SbtServerSocketHandler$$anon$1.run(SbtServerSocketHandler.scala:43)
Caused by: java.net.SocketException: Software caused connection abort: recv failed
	at java.net.SocketInputStream.socketRead0(Native Method)
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:116)
	at java.net.SocketInputStream.read(SocketInputStream.java:170)
	at java.net.SocketInputStream.read(SocketInputStream.java:141)
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246)
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265)
	at java.io.DataInputStream.readInt(DataInputStream.java:387)
	at sbt.impl.ipc.Peer.receive(IPC.scala:95)
	at sbt.impl.ipc.Peer.handshake(IPC.scala:49)
	... 2 more
Taking next connection to: 53824
New client attempting to connect on port: 59133-53824
  Address = /192.168.56.1:53824
This client on port 59133 has uuid 8fbc47ab-e9b5-4103-b48d-e21346ec54bb configName play-fork-run humanReadableName Play Fork Run protocolVersion 1 featureTags Vector()
Connected Clients: activator-44fc7ba3-7c3c-47c2-8ee4-e0bb0c2f1386, play-fork-run-8fbc47ab-e9b5-4103-b48d-e21346ec54bb
Taking next connection to: 53824
New client attempting to connect on port: 59338-53824
  Address = /192.168.56.1:53824
Handshake exception on socket: 59338
sbt.impl.ipc.HandshakeException: Unable to perform handshake
	at sbt.impl.ipc.Peer.handshake(IPC.scala:46)
	at sbt.impl.ipc.MultiClientServer.<init>(IPC.scala:150)
	at sbt.server.SbtServerSocketHandler$$anon$1.run(SbtServerSocketHandler.scala:43)
Caused by: java.net.SocketException: Software caused connection abort: recv failed
	at java.net.SocketInputStream.socketRead0(Native Method)
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:116)
	at java.net.SocketInputStream.read(SocketInputStream.java:170)
	at java.net.SocketInputStream.read(SocketInputStream.java:141)
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246)
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265)
	at java.io.DataInputStream.readInt(DataInputStream.java:387)
	at sbt.impl.ipc.Peer.receive(IPC.scala:95)
	at sbt.impl.ipc.Peer.handshake(IPC.scala:49)
	... 2 more
Taking next connection to: 53824
New client attempting to connect on port: 59339-53824
  Address = /192.168.56.1:53824
This client on port 59339 has uuid 91fb534f-24fd-45ea-ba40-32b9d5443c86 configName play-fork-run humanReadableName Play Fork Run protocolVersion 1 featureTags Vector()
Connected Clients: activator-44fc7ba3-7c3c-47c2-8ee4-e0bb0c2f1386, play-fork-run-91fb534f-24fd-45ea-ba40-32b9d5443c86
Taking next connection to: 53824
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 53824
New client attempting to connect on port: 59695-53824
  Address = /192.168.56.1:53824
Handshake exception on socket: 59695
sbt.impl.ipc.HandshakeException: Unable to perform handshake
	at sbt.impl.ipc.Peer.handshake(IPC.scala:46)
	at sbt.impl.ipc.MultiClientServer.<init>(IPC.scala:150)
	at sbt.server.SbtServerSocketHandler$$anon$1.run(SbtServerSocketHandler.scala:43)
Caused by: java.net.SocketException: Software caused connection abort: recv failed
	at java.net.SocketInputStream.socketRead0(Native Method)
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:116)
	at java.net.SocketInputStream.read(SocketInputStream.java:170)
	at java.net.SocketInputStream.read(SocketInputStream.java:141)
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246)
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265)
	at java.io.DataInputStream.readInt(DataInputStream.java:387)
	at sbt.impl.ipc.Peer.receive(IPC.scala:95)
	at sbt.impl.ipc.Peer.handshake(IPC.scala:49)
	... 2 more
Taking next connection to: 53824
New client attempting to connect on port: 59696-53824
  Address = /192.168.56.1:53824
This client on port 59696 has uuid c1510042-9cb2-4405-a5ad-fa785cea0de3 configName play-fork-run humanReadableName Play Fork Run protocolVersion 1 featureTags Vector()
Connected Clients: activator-44fc7ba3-7c3c-47c2-8ee4-e0bb0c2f1386, play-fork-run-c1510042-9cb2-4405-a5ad-fa785cea0de3
Taking next connection to: 53824
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 53824
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 53824
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 53824
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 53824
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 53824
New client attempting to connect on port: 60909-53824
  Address = /192.168.56.1:53824
Handshake exception on socket: 60909
sbt.impl.ipc.HandshakeException: Unable to perform handshake
	at sbt.impl.ipc.Peer.handshake(IPC.scala:46)
	at sbt.impl.ipc.MultiClientServer.<init>(IPC.scala:150)
	at sbt.server.SbtServerSocketHandler$$anon$1.run(SbtServerSocketHandler.scala:43)
Caused by: java.io.EOFException
	at java.io.DataInputStream.readInt(DataInputStream.java:392)
	at sbt.impl.ipc.Peer.receive(IPC.scala:95)
	at sbt.impl.ipc.Peer.handshake(IPC.scala:49)
	... 2 more
Taking next connection to: 53824
New client attempting to connect on port: 60910-53824
  Address = /192.168.56.1:53824
This client on port 60910 has uuid 0c0803fe-33ea-44e6-a6ab-c89b4e529fdc configName play-fork-run humanReadableName Play Fork Run protocolVersion 1 featureTags Vector()
Connected Clients: activator-44fc7ba3-7c3c-47c2-8ee4-e0bb0c2f1386, play-fork-run-0c0803fe-33ea-44e6-a6ab-c89b4e529fdc
Taking next connection to: 53824
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 53824
New client attempting to connect on port: 61312-53824
  Address = /192.168.56.1:53824
Handshake exception on socket: 61312
sbt.impl.ipc.HandshakeException: Unable to perform handshake
	at sbt.impl.ipc.Peer.handshake(IPC.scala:46)
	at sbt.impl.ipc.MultiClientServer.<init>(IPC.scala:150)
	at sbt.server.SbtServerSocketHandler$$anon$1.run(SbtServerSocketHandler.scala:43)
Caused by: java.io.EOFException
	at java.io.DataInputStream.readInt(DataInputStream.java:392)
	at sbt.impl.ipc.Peer.receive(IPC.scala:95)
	at sbt.impl.ipc.Peer.handshake(IPC.scala:49)
	... 2 more
Taking next connection to: 53824
New client attempting to connect on port: 61313-53824
  Address = /192.168.56.1:53824
This client on port 61313 has uuid 52d92762-6f9a-4bbd-877a-4285e8584d33 configName play-fork-run humanReadableName Play Fork Run protocolVersion 1 featureTags Vector()
Connected Clients: activator-44fc7ba3-7c3c-47c2-8ee4-e0bb0c2f1386, play-fork-run-0c0803fe-33ea-44e6-a6ab-c89b4e529fdc, play-fork-run-52d92762-6f9a-4bbd-877a-4285e8584d33
Taking next connection to: 53824
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 53824
New client attempting to connect on port: 61732-53824
  Address = /192.168.56.1:53824
Handshake exception on socket: 61732
sbt.impl.ipc.HandshakeException: Unable to perform handshake
	at sbt.impl.ipc.Peer.handshake(IPC.scala:46)
	at sbt.impl.ipc.MultiClientServer.<init>(IPC.scala:150)
	at sbt.server.SbtServerSocketHandler$$anon$1.run(SbtServerSocketHandler.scala:43)
Caused by: java.io.EOFException
	at java.io.DataInputStream.readInt(DataInputStream.java:392)
	at sbt.impl.ipc.Peer.receive(IPC.scala:95)
	at sbt.impl.ipc.Peer.handshake(IPC.scala:49)
	... 2 more
Taking next connection to: 53824
New client attempting to connect on port: 61733-53824
  Address = /192.168.56.1:53824
This client on port 61733 has uuid b6ea68d8-3e73-4c4d-bbc8-770b4ceee732 configName play-fork-run humanReadableName Play Fork Run protocolVersion 1 featureTags Vector()
Connected Clients: activator-44fc7ba3-7c3c-47c2-8ee4-e0bb0c2f1386, play-fork-run-b6ea68d8-3e73-4c4d-bbc8-770b4ceee732
Taking next connection to: 53824
New client attempting to connect on port: 61738-53824
  Address = /192.168.56.1:53824
Handshake exception on socket: 61738
sbt.impl.ipc.HandshakeException: Unable to perform handshake
	at sbt.impl.ipc.Peer.handshake(IPC.scala:46)
	at sbt.impl.ipc.MultiClientServer.<init>(IPC.scala:150)
	at sbt.server.SbtServerSocketHandler$$anon$1.run(SbtServerSocketHandler.scala:43)
Caused by: java.io.EOFException
	at java.io.DataInputStream.readInt(DataInputStream.java:392)
	at sbt.impl.ipc.Peer.receive(IPC.scala:95)
	at sbt.impl.ipc.Peer.handshake(IPC.scala:49)
	... 2 more
Taking next connection to: 53824
New client attempting to connect on port: 61739-53824
  Address = /192.168.56.1:53824
This client on port 61739 has uuid 9781592f-b7d0-4d7e-b93b-611908f10d02 configName play-fork-run humanReadableName Play Fork Run protocolVersion 1 featureTags Vector()
Connected Clients: activator-44fc7ba3-7c3c-47c2-8ee4-e0bb0c2f1386, play-fork-run-b6ea68d8-3e73-4c4d-bbc8-770b4ceee732, play-fork-run-9781592f-b7d0-4d7e-b93b-611908f10d02
Taking next connection to: 53824
New client attempting to connect on port: 61799-53824
  Address = /192.168.56.1:53824
Handshake exception on socket: 61799
sbt.impl.ipc.HandshakeException: Unable to perform handshake
	at sbt.impl.ipc.Peer.handshake(IPC.scala:46)
	at sbt.impl.ipc.MultiClientServer.<init>(IPC.scala:150)
	at sbt.server.SbtServerSocketHandler$$anon$1.run(SbtServerSocketHandler.scala:43)
Caused by: java.net.SocketException: Software caused connection abort: recv failed
	at java.net.SocketInputStream.socketRead0(Native Method)
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:116)
	at java.net.SocketInputStream.read(SocketInputStream.java:170)
	at java.net.SocketInputStream.read(SocketInputStream.java:141)
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246)
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265)
	at java.io.DataInputStream.readInt(DataInputStream.java:387)
	at sbt.impl.ipc.Peer.receive(IPC.scala:95)
	at sbt.impl.ipc.Peer.handshake(IPC.scala:49)
	... 2 more
Taking next connection to: 53824
New client attempting to connect on port: 61800-53824
  Address = /192.168.56.1:53824
This client on port 61800 has uuid 48d8e558-b56c-4908-b6ba-1a74d6cb7254 configName play-fork-run humanReadableName Play Fork Run protocolVersion 1 featureTags Vector()
Connected Clients: activator-44fc7ba3-7c3c-47c2-8ee4-e0bb0c2f1386, play-fork-run-48d8e558-b56c-4908-b6ba-1a74d6cb7254
Taking next connection to: 53824
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 53824
